pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -203,6 +203,9 @@ type PipelineRef struct {
 	// API version of the referent
 	// +optional
 	APIVersion string `json:"apiVersion,omitempty"`
+	// Bundle url reference to a Tekton Bundle.
+	// +optional
+	Bundle string `json:"bundle,omitempty"`
```

Comment:
Do we want to prefix these bundles with something like `oci://<url>` in case we want to support other transports later?

Reply:
I think that makes sense. I am also thinking a lot about how this expands to other references in the future. In general, I am hesitant to use non-standard schemes since it means brittle parsing logic. My preference here is to punt on this question since the field `type` probably doesn't need to change but we can change the validation out of band.

Reply:
I definitely agree with @pierretasci, I am not a huge fan of using non-standard schemes, or schemes at all in image refs, so having specific fields feels safer :angel: